addzopeuser accepts custom location of zope.conf (#1067)
zopefoundation/Zope: docs/operation.rst

  $ bin/addzopeuser user password

The script expects to find the configuration file at ``etc/zope.conf`` by default.

> If it is located on a different location this can be specified by the `--configuration` option:

Reviewer: ```suggestion
If it is located in a different location you can specify it with the `--configuration` option:
```
